front-end/src/components/HelpData.js

```diff
@@ -0,0 +1,121 @@
+// probably too verbose to put in the actual code
```

Comment:
This whole file should be used INSTEAD of what we have in Analyzerd and Series, to avoid redundancy and updating affecting all locations of its call